docs/examples/pytorch/single_stage_detector/LICENSE

```diff
@@ -185,6 +185,8 @@
       file or class name and description of purpose be included on the
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
+   
+   Copyright 2018 NVIDIA Corporation
```

Comment:
The whole code base was written solely in 2018?

Reply:
Done